boofuzz/__init__.py

```diff
@@ -365,14 +365,14 @@ def s_block_end(name=None):
 
 
 def s_checksum(
-        block_name,
-        algorithm="crc32",
-        length=0,
-        endian=LITTLE_ENDIAN,
-        fuzzable=True,
-        name=None,
-        ipv4_src_block_name=None,
-        ipv4_dst_block_name=None,
+    block_name,
+    algorithm="crc32",
+    length=0,
+    endian=LITTLE_ENDIAN,
+    fuzzable=True,
+    name=None,
+    ipv4_src_block_name=None,
+    ipv4_dst_block_name=None,
```

Comment:
@SR4ven This is one place where the PyCharm "Reformat File" behavior differs from Stickler-CI's. Reformat File makes all these lines double-indented. Sticker-CI prefers a single indentation. Doesn't really matter, but makes it easy to end up with a lot of stickler-ci changes.

Reply:
Usually I only use PyCharm to optimize imports and then `black .` to reformat.
Running `tox` will also fail at the lint check if black finds something to reformat.

I think it's possible to add a PyCharm project config file to git which might allow us to unify differences between PyCharm and black. I'm not sure how many differences there are and if it's worth the time.

The specific issue you mentioned can be fixed at Settings > Editor > Code Style > Python > Continuation Indent. Another option would be to ignore the PyCharm reformatting and use the [black integration](https://black.readthedocs.io/en/stable/editor_integration.html) instead.

Reply:
Thanks! I changed that PyCharm setting for now.